### PR TITLE
Added acronym command

### PR DESCRIPTION
**Pull request by CameronAavik**

### CODE REVIEW

**Comment by CameronAavik** [comment]
wat, my branch is up to date with no conflicts?

**Comment by CameronAavik** [comment]
Nvm, it was just because it didn't want to rebase

File: uqcsbot/scripts/acronym.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        if word.lower() in [":horse:", "horse"]:
+            bot.post_message(command.channel, ">:taco:")
+            return
+        if word.lower() in [":rachel:", "rachel"]:
+            bot.post_message(command.channel, ">:older_woman:")
+            return
+
+    loop = asyncio.get_event_loop()
+    acronym_futures = []
+    for word in words[:ACRONYM_LIMIT]:
```
Comment (TRManderson) [inline uqcsbot/scripts/acronym.py]: Could be replaced with a list comprehension, but good as is

File: uqcsbot/scripts/acronym.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+
+@bot.on_command("acro")
+async def handle_acronym(command: Command):
+    if not command.has_arg():
+        return
+
+    words = command.arg.split(" ")
+
+    if len(words) == 1:
```
Comment (TRManderson) [inline uqcsbot/scripts/acronym.py]: Can you readd the comment from the JS here?

File: uqcsbot/scripts/acronym.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+            bot.post_message(command.channel, ">:taco:")
+            return
+        if word.lower() in [":rachel:", "rachel"]:
+            bot.post_message(command.channel, ">:older_woman:")
+            return
+
+    loop = asyncio.get_event_loop()
+    acronym_futures = []
+    for word in words[:ACRONYM_LIMIT]:
+        get_task = loop.run_in_executor(None, get, f"{BASE_URL}/{quote(word)}")
```
Comment (TRManderson) [inline uqcsbot/scripts/acronym.py]: You can just do `bot.run_async(get, f"{BASE_URL}/{quote(word)}")`

**Pull request merged**